src/renderer/components/player-settings/player-settings.scss

```diff
@@ -0,0 +1,18 @@
+.screenshotFolderContainer {
+  width: 95%;
+  margin: 0 auto;
+  align-items: center;
+  column-gap: 1rem;
+
+  .screenshotFolderLabel,
+  .screenshotFolderButton,
+  .screenshotFilenamePatternTitle {
+    flex-grow: 0;
+  }
+  .screenshotFolderPath,
```

Comment:
Could you please make the spacing consistent, either with empty lines between all of the rules (easier to read) or remove the empty lines. Currently you've got a mix between both in different and even with the same files.

Reply:
Done